app/models/metadatum.rb

```diff
@@ -247,6 +253,21 @@ def self.bulk_load_import(to_create)
     errors
   end
 
+  # Bulk import safe to use with elasticsearch
+  def self.bulk_import_safe(to_create, opts = {})
```

Comment:
why not just override `bulk_import`? it seems like that would be the safest way to ensure good behavior for idseq